client/my-sites/sharing/connections/service-description.jsx

```diff
@@ -148,6 +148,17 @@ class SharingServiceDescription extends Component {
 					}
 				);
 			},
+			google_my_business: function() {
+				if ( this.props.numberOfConnections > 0 ) {
+					return this.props.translate( 'Manage your connected Google My Business locations', {
```

Comment:
Please ignore, this will be removed when https://github.com/Automattic/wp-calypso/pull/23638 is merged